docs/charts/roc_chart_from_labels_table.ipynb

```diff
@@ -172,15 +169,39 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### What the chart shows\n"
+    "### What the chart shows\n",
+    "\n",
+    "The chart plots the True Positive Rate against False Positive Rate for clerically reviewed records. Each point on the curve reflects the choice of a match weight threshold for a match and the subsequent True/False Positive Rates.\n",
+    "\n",
+    "??? note \"What the chart tooltip shows\"\n",
+    "    ![](./img/roc_chart_from_labels_table_tooltip.png)\n",
+    "\n",
+    "    The tooltip shows information based on the point on the curve that the user is hoverng over, including:\n",
```

Comment:
A quick aside, should we adjust the `truth_threshold` chart label? I understand why it's been named as such, but it's not the clearest description.